resources/core/charts/helm-broker/values.yaml

```diff
@@ -8,19 +8,10 @@ service:
   externalPort: 80
   internalPort: 8080
 
-embeddedRepository:
-  # Defines, if embedded bundle repository should be provisioned.
-  # To provision, specify this value to true
-  provision: true
-  # Defines service for embedded repository
-  service:
-    type: NodePort
-    externalPort: 80
-    internalPort: 8080
 
 config:
   repository:
-    baseURL: "http://core-helm-broker-reposerver"
+    baseURL: "https://kyma-project.github.io/kyma-bundles/stable/"
```

Comment:
to: https://kyma-project.github.io/bundles/stable/

Reply:
changed